### PR TITLE
feat(Modal): remove modal max height 

### PR DESCRIPTION
**Pull request by MkMan**
These changes are to remove the max-height restriction on the modal (as per the UX team's request).

## Before
before

## After
after

### CODE REVIEW

File: src/components/Modal/Modal.stories.mdx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import * as knobs from '@storybook/addon-knobs';
 import { State } from '~.storybook/State';
 import { Button } from '~/components/Button';
 import { Modal } from './Modal';
-import { ModalFixture } from './Modal.fixture';
+import { ModalFixture } from './Modal.fixture.tsx';
```
Comment (ivandaho) [inline src/components/Modal/Modal.stories.mdx:6]: Is the .tsx extension needed?

Comment (MkMan) [reply src/components/Modal/Modal.stories.mdx:6]: I could have swore that Storybook complained when it wasn't there but I just removed it and it works fine. I guess it's not needed, I'll remove it. Thank you 😃

Comment (MkMan) [reply src/components/Modal/Modal.stories.mdx]: Done ✔️

**Comment by cyrusxzw** [comment]
Looks good, but may need to check the component using Modal, make sure they are still working well. 😄

**Review by cyrusxzw — APPROVED** [review]

**Pull request merged**